app/views/posts/index.html.erb

```diff
@@ -1 +1,37 @@
-<h1>Route for list of post<h1>
\ No newline at end of file
+ <head>
+  <%= stylesheet_link_tag 'application.css' %>
+</head>
+
+<div>
+  <div class="parent-container">
+    <div class="user-image">
+      <img src=<%= @user.photo %> width="75rem">
+    </div>
+
+    <div class="user-details">
+      <h1><%= @user.name %></h1>
+      <p>Number of post: <%= @user.posts_counter %></p>
+    </div>
+  </div>
+
+  <% @user.posts.each do |post| %>
+  <div class="post">
+    <h2>Post: <%= link_to post.id, user_posts_path(@user, post) %></h2>
```

Comment:
- [ Required ] Good job so far, I think it would be better if you made this link work correctly. Currently, when I click it doesn't open the posts show page. I would like to suggest that you refactor this line as shown below, so that a user gets the correct page when they click on the link.

```ruby
<h2>Post: <%= link_to post.id, user_post_path(@user.id, post.id) %></h2>
```